ld/ldexp.c

```diff
@@ -1364,6 +1364,7 @@ static etree_type *
 exp_assop (const char *dst,
 	   etree_type *src,
 	   enum node_tree_enum class,
+	   bool defsym,
```

Comment:
doesn't look microblaze related: to be inspected

Reply:
Revert "ld: Remove unused expression state" --defsym symbol=expression Create a global symbol in the output file, containing the absolute address given by expression.

This reverts commit 65f14869fd3fbee8ed4c4ca49de8aaa86dbc66cb.

Reply:
Part of
https://github.com/Xilinx/meta-xilinx/blob/master/meta-microblaze/recipes-devtools/binutils/binutils/0021-Revert-ld-Remove-unused-expression-state.patch 
which brings back `defsym` variables which is used in 
https://github.com/Xilinx/meta-xilinx/blob/master/meta-microblaze/recipes-devtools/binutils/binutils/0021-Revert-ld-Remove-unused-expression-state.patch
Relates to: https://github.com/zephyrproject-rtos/binutils-gdb/pull/10#discussion_r1421607474
Keeping it in.